database/buildSchema/02_user.sql

```diff
@@ -10,3 +10,7 @@ CREATE TABLE public.user (
     password_hash varchar
 );
 
+-- adds citext constraint to username for case insensitivity
+CREATE EXTENSION IF NOT EXISTS citext;
+ALTER TABLE public.user ALTER COLUMN username TYPE citext;
+
```

Comment:
You should be able to put `CREATE EXTENSION IF NOT EXISTS citext;` at the top of the file and then just define the type directly:

```
CREATE TABLE public.user (
    id serial PRIMARY KEY,
    ...
    username citext UNIQUE,
    email varchar,
    ...
);
```